<launch>
  <!-- Turtlesim Node-->
  <node pkg="turtlesim" type="turtlesim_node" name="simulator"/>
  <node pkg="turtlesim" type="turtle_teleop_key" name="keyboard" output="screen"/>
  <!-- Axes -->
  <node name="turtle1_tf_broadcaster" pkg="study_pkg" type="turtle_tf_broadcaster.py" respawn="false" output="screen">
    <remap from="input_pose" to="turtle1/pose"/>
    <param name="turtle_tf_broadcaster" type="string" value="turtle1"/>
  </node>
  <node name="turtle2_tf_broadcaster" pkg="study_pkg" type="turtle_tf_broadcaster.py" respawn="false" output="screen">
    <remap from="input_pose" to="turtle2/pose"/>
    <param name="turtle_tf_broadcaster" type="string" value="turtle2"/>
  </node>
  <node name="turtle_tf_listener" pkg="study_pkg" type="turtle_tf_listener.py" respawn="false" output="screen"/>
  <!--<node pkg="tf" type="static_transform_publisher" name="car_2_camera_tf" args="0.3 0.8 0.2 0 0 0 turtle1 carrot 100" />-->
  <node pkg="rviz" type="rviz" name="rviz" />
</launch>